File: BOJ/java/1476.java
import java.io.IOException;
import java.util.Scanner;

class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int e = sc.nextInt() - 1;
        int m = sc.nextInt() - 1;
        int s = sc.nextInt() - 1;

        for (int i = 0; ; i++) {
            if (i % 15 == e && i % 28 == m && i % 19 == s) {
                System.out.println(i + 1);
                sc.close();
                System.exit(0);
            }
        }
    }
}

File: BOJ/kotlin/1476.kt
import java.util.*
import kotlin.system.exitProcess

fun main() = with(Scanner(System.`in`)) {
    val e = nextInt() - 1
    val m = nextInt() - 1
    val s = nextInt() - 1

    var i = 0
    while (true) {
        if (i % 15 == e && i % 28 == m && i % 19 == s) {
            println(i + 1)
            exitProcess(0)
        }
        i++
    }
}